webapp/src/pages/welcome/welcomePage.test.tsx

```diff
@@ -8,19 +8,28 @@ import {createMemoryHistory} from 'history'
 
 import {Router} from 'react-router-dom'
 
+import {Provider as ReduxProvider} from 'react-redux'
+
 import userEvent from '@testing-library/user-event'
 
-import {UserSettings} from '../../userSettings'
+import configureStore from 'redux-mock-store'
 
 import {wrapIntl} from '../../testUtils'
 
+import {FetchMock} from '../../test/fetchMock'
+
 import WelcomePage from './welcomePage'
 
+import 'isomorphic-fetch'
+
 const w = (window as any)
 const oldBaseURL = w.baseURL
 
+global.fetch = FetchMock.fn
+
 beforeEach(() => {
-    UserSettings.welcomePageViewed = null
+    FetchMock.fn.mockReset()
+    FetchMock.fn.mockReturnValue(FetchMock.jsonResponse(JSON.stringify({})))
```

Comment:
May be mocking `mutator` like in many other unit tests will be a little bit better.

Reply:
@kamre I'm having a hard time finding where we are mocking `mutator`. Can you share one such example?

Reply:
Ah, got it! Thanks for pointing that out. That definitely looks much better than emulating fetch. I'll make the change 👍

Reply:
Done.